qemu_v8/qemu-v8-scmi-overlay.dtso

```diff
@@ -0,0 +1,50 @@
+/dts-v1/;
+/plugin/;
+
+&{/} {
+	cpus {
+		cpu@0 {
+			clocks = <&scmi_dvfs 0>;
+		};
+
+		cpu@1 {
+			clocks = <&scmi_dvfs 0>;
+		};
+	};
+
+	firmware {
+		optee {
+			compatible = "linaro,optee-tz";
+			method = "smc";
+		};
+
+		scmi: scmi0 {
+			compatible = "linaro,scmi-optee";
+			#address-cells = <1>;
+			#size-cells = <0>;
+
+			linaro,optee-channel-id = <0x01 0x04>;
```

Comment:
2 channel IDs?
Which refers to what?

Reply:
The 2nd is the notification channel (not supported yet by optee)